recipes/daetk/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "daetk" %}
+{% set version = "1.0.1" %}
+{% set build = 0 %}
+{% set sha256 = "5080ab702ea21987b1e6574e8506ff7d48071a5007d8a1bce1a4bd480e659c15" %}
+
+{% set mpi = mpi or 'mpich' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: daetk-{{ version }}.tar.gz
+  url: https://github.com/erdc-cm/daetk/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - edit_content.patch
+
+build:
+  number: {{ build }}
+  skip: True  # [win]
+  {% set mpi_prefix = "mpi_" + mpi %}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+  run_exports:
+    - {{ name }} * {{ mpi_prefix }}_*
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - sed
+  host:
+    - petsc
+    - parmetis
+    - {{ mpi }}
+  run:
+    # missing a run_exports section
+    - {{ pin_compatible('parmetis', max_pin='x.x') }}
+    # these are second-order dependencies that lack a run_exports section
+    - hypre
+    - scotch
+    - scalapack
+    - ptscotch
+    - metis
+    - suitesparse
+    - mumps-mpi
+    - libopenblas
+    - conda-build
+
+test:
+  commands:
+    - conda inspect linkages daetk  # [not win]
+
+about:
+  home: https://github.com/erdc-cm/daetk
+  license: MIT
+  license_file: LICENSE.md
+  summary: The Differential Algebraic Equation Toolkit
+
+extra:
+  recipe-maintainers:
+    - davidbrochart
+    - SylvainCorlay
```

Comment:
@SylvainCorlay 
Do you agree to maintain this recipe as well?

Reply:
Absolutely.